hal/halx86/acpi.cmake

```diff
@@ -1,11 +1,19 @@
 
+include_directories(include ${REACTOS_SOURCE_DIR}/drivers/bus/acpi/acpica/include)
+
 list(APPEND HAL_ACPI_SOURCE
     acpi/halacpi.c
     acpi/halpnpdd.c
     acpi/busemul.c
+    acpi/madt.c
     legacy/bus/pcibus.c)
 
+if(ARCH STREQUAL "amd64")
+    add_definitions(-DWIN64)
```

Comment:
Isn't this thing automatically defined at a global level? (Also that looks like a win32-user-mode-only define, not kernel-mode... to be verified with other kernel-mode code + headers...)

Reply:
It should be but i guess it's not with the HALs it super breaks compilation with ACPICA when i remove it

Reply:
ok my advice here would be to add a comment that this is needed for acpica.